backend/cfg/cfg_intf.ml

```diff
@@ -40,18 +40,29 @@ module S = struct
     { func_symbol : string;
       alloc : bool;
       (* CR mshinwell: rename [alloc] -> [needs_caml_c_call] *)
+      returns : Label.t option;
+          (** At least one of [alloc] and [returns] must be set.  Otherwise,
```

Comment:
> At least one of [alloc] and [returns] must be set

Not necessarily in this pull request, but shouldn't
we enforce that through types?

Reply:
See cmm.mli